app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/TitleAndDescription.tsx

```diff
@@ -1,74 +1,96 @@
+"use client";
+import React, { useState } from "react";
 import { VaultStatus } from "@lib/types";
 import { useTranslation } from "@i18n/client";
+import { ConfirmDialog } from "./Dialogs/ConfirmDialog";
+import { usePathname, useRouter } from "next/navigation";
+import { Alert } from "@clientComponents/globals";
 
 export const TitleAndDescription = ({
   statusFilter,
-  setShowConfirmReceiptDialog,
+  formId,
+  responseDownloadLimit,
 }: {
   statusFilter: string;
-  setShowConfirmReceiptDialog: (show: boolean) => void;
+  formId: string;
+  responseDownloadLimit: number;
 }) => {
   const { t } = useTranslation("form-builder-responses");
+  const [successAlertMessage, setShowSuccessAlert] = useState<false | string>(false);
+  const router = useRouter();
+  const pathName = usePathname();
   return (
     <>
-      <>
-        {statusFilter == VaultStatus.NEW && (
-          <>
-            <h1>{t("tabs.newResponses.title")}</h1>
-            <div className="mb-4">
-              <p className="mb-4">
-                <strong>{t("tabs.newResponses.message1")}</strong>
-                <br />
-                {t("tabs.newResponses.message2")}
-              </p>
-            </div>
-          </>
-        )}
-        {statusFilter === VaultStatus.DOWNLOADED && (
-          <>
-            <h1>{t("tabs.downloadedResponses.title")}</h1>
-            <div className="mb-4">
-              <p className="mb-4">
-                <strong>{t("tabs.downloadedResponses.message1")}</strong>
-                <br />
-                {t("tabs.downloadedResponses.message2")}
-              </p>
-              <Button onClick={() => setShowConfirmReceiptDialog(true)} theme="secondary">
-                {t("responses.confirmReceipt")}
-              </Button>
-            </div>
-          </>
-        )}
-        {statusFilter === VaultStatus.CONFIRMED && (
-          <>
-            <h1>{t("tabs.confirmedResponses.title")}</h1>
-            <div className="mb-4">
-              <p className="mb-4">
-                <strong>{t("tabs.confirmedResponses.message1")}</strong>
-                <br />
-                {t("tabs.confirmedResponses.message2")}
-              </p>
-            </div>
-          </>
-        )}
-        {statusFilter === VaultStatus.PROBLEM && (
-          <>
-            <h1>{t("tabs.problemResponses.title")}</h1>
-            <div className="mb-4">
-              <p className="mb-4">
-                <strong>{t("tabs.problemResponses.message1")}</strong>
-                <br />
-                {t("tabs.problemResponses.message2")}
-              </p>
-              <Button onClick={() => setShowConfirmReceiptDialog(true)} theme="secondary">
-                {t("responses.confirmReceipt")}
-              </Button>
-            </div>
-          </>
-        )}
-      </>
+      {statusFilter == VaultStatus.NEW && (
+        <>
+          <h1>{t("tabs.newResponses.title")}</h1>
+          <div className="mb-4">
```

Comment:
Looks like we're doubling up in the mb-4 one here + one on the inner p (which I think collapses ) ... can remove one on the p tag?